src/wristpy/processing/metrics.py

```diff
@@ -0,0 +1,27 @@
+"""Calculate base metrics, anglez and enmo."""
+
+import numpy as np
+
+from wristpy.core.models import Measurement
+
+
+def euclidean_norm(acceleration: Measurement)-> Measurement:
```

Comment:
I know you copied this from the diagram; so this is my bad, but since this isn't strictly a euclidean norm I wouldn't use this name. 

How about we name this `def euclidean_norm_minus_one` or `def euclidean_norm_min_one` if you want to shorten it a bit. Feel free to use anything else as long as the functionality is self-evident from the name.